google-cloud-tools-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -137,7 +139,8 @@ public void actionPerformed(ActionEvent e) {
     }
     BasicAction.saveAll();
 
-    wireup = new ProjectDebuggeeBinding(elysiumProjectSelector, debuggeeTarget);
+    this.wireup = wireup == null ?
```

Comment:
Unless there is a strong opinion on this, I'm going to leave this specific case with the ternary operator just because the conditional is being used for assignment and the ternary (in my opinion) makes it slightly cleaner since it returns a value.